ipatests/test_integration/test_cert.py

```diff
@@ -127,6 +128,84 @@ def test_ipa_getcert_san_aci(self):
         ipaddrs = ext.value.get_values_for_type(x509.IPAddress)
         assert ipaddrs == [ipaddress.ip_address(self.clients[0].ip)]
 
+    def test_getcert_list_profile(self):
+        """
+        Test that getcert list command displays the profile
+        for the requests
+        """
+        for requests in os.listdir(paths.CERTMONGER_REQUESTS_DIR):
```

Comment:
`os.listdir()` will be run on runner and not on `master`.

Reply:
@mrizwan93 the code was removed.